packages/@ama-sdk/create/src/index.ts

```diff
@@ -65,6 +65,11 @@ if (argv['spec-path'] && argv['spec-package-name']) {
   process.exit(-4);
 }
 
+if (argv['spec-package-name'] && !argv['spec-package-path']) {
```

Comment:
Not sure it is a good idea to enforce `spec-package-path` in every case mainly because we generate the artifact containing the specification so we should be able to retrieve it in most of the cases.
Forcing the user to provide the path to a yarm within a package containing only one yaml file and for which the user does not the structure seems a bit too much.

For me a fix can be:
- To align the structure of the artifact (generated by the swagger-builder today may that we should define in doc) with the this part of the code determining the default spec file
- Displaying a warning with the default spec file used
- If no spec file available, throwing with this error

Reply:
I think it's a good idea to default on a path. I am not so sure about the structure, because I don't know how much the swagger-builder will be used. But as I see no other options, let's use that one.